### PR TITLE
Release Github Actions

### PR DESCRIPTION
**Pull request by ah1053**
1. Publish docker image to MobiledgeXSamples Dockerhub.
2. Push docker image to MobiledgeX registery and deploy app instances in EU,US & JP.

### CODE REVIEW

**Review by brucearmstrong — APPROVED** [review]

File: .mobiledgex/appinsts.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+- appinst:
+    key:
+      cluster_inst_key:
+        cluster_key:
+          name: autoclusteredgemultiplay
```
Comment (franklin-huang-mobiledgex) [inline .mobiledgex/appinsts.yml:5]: will this always be the same cluster name regardless of where it is deployed?

**Review by franklin-huang-mobiledgex — APPROVED** [review]

File: .mobiledgex/appinsts.yml (hunk 1 above)
Comment (ah1053) [reply .mobiledgex/appinsts.yml:5]: autoclusteredgemultiplay we have this cluster in the 3 regions

**Review by venkytv — APPROVED** [review]
Looks great, Ahmed! Thanks!

I have an update pending for a new version of the Github Action (https://github.com/mobiledgex/deploy-app-action/releases/tag/v1.2). Once that is released, we could change the version to that. It is backwards compatible and should not need any other changes.

**Review by lgarner — APPROVED** [review]
lgtm

**Pull request merged**